certs/sp/mymovemil.crt

```diff
@@ -0,0 +1 @@
+-----BEGIN CERTIFICATE----- MIIDkjCCAnoCCQDqY3DWYAKhZDANBgkqhkiG9w0BAQsFADCBijEcMBoGA1UEAwwT aHR0cHM6Ly9teS5tb3ZlLm1pbDEMMAoGA1UECgwDRG9EMQwwCgYDVQQLDANERFMx EzARBgNVBAcMCldhc2hpbmd0b24xCzAJBgNVBAgMAkRDMQswCQYDVQQGEwJVUzEf MB0GCSqGSIb3DQEJARYQdHJhbnNjb21AZGRzLm1pbDAeFw0xODAyMDcyMTU0MTZa Fw0xOTAyMDcyMTU0MTZaMIGKMRwwGgYDVQQDDBNodHRwczovL215Lm1vdmUubWls MQwwCgYDVQQKDANEb0QxDDAKBgNVBAsMA0REUzETMBEGA1UEBwwKV2FzaGluZ3Rv bjELMAkGA1UECAwCREMxCzAJBgNVBAYTAlVTMR8wHQYJKoZIhvcNAQkBFhB0cmFu c2NvbUBkZHMubWlsMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA4Iob rWfTYlvIexLKerJxEnLVCMJQydzk0aIfaDSM8Z9rRuZsd1A/9VqnO7zOfohc0F4n 1yAQfH2fzoE+OdKEuMjhmVjuPQCO3Qje0svyQsvXV+cLSb78381tQH0vvXZKINJ0 uLjeLprA54sEQGWJyhuFo9lzk4a4Y5fxjEHulrUuuFeAIznX0HV9xfW3kWGn9Pq5 klzYbUm+xAow9Eo9y5HTxkxqEax54O4EhnSorSiE0tZDZP0AloAT9VvGkHYumygn qbYp6SdIRyYBgGTVM6/KjpKECcQsyVNPwdZc8U8S3qtJyt22GPxhU8NduSkBYxNZ Kty2vGqrmXLqGa7k+QIDAQABMA0GCSqGSIb3DQEBCwUAA4IBAQCsIdzquPPJCL3l wXkC3OhMHC/N7gjF0EQrCz4/VZZMIUYUQXzl76RRh1Z/nfu7Ji21YIXreSpygwhk yu33LCXFiykCI8m5Um//d1BIICzcuH1aZXo4J2h6I9CLrWqIg+oO3HyyX0eHSvZe 6AM+GV/BrxWDH6bsu2UMo2ml3xit+RM1rrBRu8kOj4CpGYwqj9lvLwsAOhZIGNkP husVDMyH79Rq8+bS9Mp4F1kGGlf2Zzs2CsX9aWJvCp9ywitwhvPz9H2uZA36OWyw jYrecqVUYf8EQrw6kssgvfAXUT4lvjhE6qtVms2e3oV/btM5RChHvtlioI1Y1K5o ofacSdHB -----END CERTIFICATE-----
```

Comment:
This single-line format came from the dashboard's front-end interface. 

I have asked the client developer for a version that more closely resembles the usual format.

Reply:
Actually, I was able to use browser inspect element on the identity dashboard's sp show page to get the cert's raw formatting.

Afterwards I noticed the single-line version contained space characters that would have allowed me to reconstruct the clean formatting. 😸